docs/project/roadmap.md

```diff
@@ -33,28 +33,66 @@ performance BLAS library such as BLIS.
 
 See issue {issue}`1120`.
 
-## Simplification of the package loading system
+## Improvements to package loading system
 
 Currently, Pyodide has two ways of loading packages:
 
-- `loadPackage` for packages built with Pyodide and
-- `micropip.install` for pure Python packages from PyPI.
+- {any}`pyodide.loadPackage` for packages built with Pyodide and
+- {any}`micropip.install` for pure Python packages from PyPI.
 
-The relationship between these tools is confusing and could be simplified.
-Furthermore, the dependency resolution logic and packaging / build system could
-be improved.
+The relationship between these tools is currently confusing. 
 
-See issues {issue}`1470` and {issue}`1100`.
+Our goal is to have three ways to load packages: one with no dependency
+resolution at all, one with static dependency resolution which is done ahead of
+time, and one for dynamic dependency resolution. Ideally most applications can
+use static dependency resolution and repls can use dynamic dependency
+resolution. The no-dependency approach can be used in situations where our
+built-in logic is not helpful.
 
-## Update SciPy to a more recent version
+The plan is for {any}`pyodide.loadPackage` to do static dependency resolution
```

Comment:
I think this gives too many specific details on the implementation. The roadmap is there to state what we want to achieve not how we are going to do it. Otherwise, the implementation can still get iterated on and this will become outdated. 

So I think we can remove this paragraph.